src/RandomNumberGenerator.md

```diff
@@ -1,17 +1,128 @@
 # Random Number Generator
+
 Generates a random number within a given range
 
 ## Result
+
 ```js
-  randomNumber(1, 100); // 68
+randomNumber(1, 100); // 68
 ```
 
 ## Snip Langauge
-* [JavaScript](#javascript)
+
+- [JavaScript](#javascript)
+- [Cpp](#cpp)
+- [Python](#python)
+- [Java](#java)
+- [C#](#c)
+- [Go](#go)
+- [php](#php)
+- [perl](#perl)
 
 ### JavaScript
+
 ```js
-  function randomNumber(min, max) { 
-      return Math.floor(Math.random() * (max - min-1) + min+1);
-  } 
+function randomNumber(min, max) {
+  return Math.floor(Math.random() * (max - min - 1) + min + 1);
+}
+```
+
+### Cpp
+
+```cpp
+const unsigned long long a = 6364136223846793005ULL;
+const unsigned long long c = 1ULL << 30;
+const unsigned long long m = (1ULL << 62) - 1;
```

Comment:
Keep this inside function, so the whole function can be copied easily

Reply:
done and updated some fixes